database/src/pool/postgres.rs

```diff
@@ -1525,6 +1525,42 @@ where
 
         Ok(())
     }
+
+    async fn attach_shas_to_try_benchmark_request(
+        &mut self,
+        pr: u32,
+        sha: &str,
+        parent_sha: &str,
+    ) -> anyhow::Result<()> {
+        let tx = self
```

Comment:
Why he transaction? We only run a single SQL statement, that is essentially a mini-transaction anyway, right?